### ISSUE TITLE
Support multiple root folders and etc

### ISSUE THREAD

**Issue by yxlao** [QinMing/simple-virtualenv-wrapper#13]
Some todos:
- [ ] Support multiple virtualenv root folders.
- [ ] Hidden directory names e.g. `.venv`.
- [ ] Virtualenv and other code are in the same directory.

Those are quite useful in my scenario. Are they suitable for the repo? If yes, I'll add them when I get some time.

**Comment by QinMing** [comment]
Yes that would be a nice to have. But how about simply set `VENV_ROOT=$PWD\.venv`. ? Does this handle your use case ?

**Comment by yxlao** [comment]
No as `.venv` in my case is a the actual venv dir (i.e. it contains `bin`, `lib` ...), not the dir containing venv dirs. If I set the root to be `$PWD`, then it would be mixed with other files / dirs in `$PWD`.

**Comment by QinMing** [comment]
This is actually a hard problem, feels like

**Comment by QinMing** [comment]
How about the above commit? @yxlao 
If you do this way, do you have to gitignore the folder for every single project (or .git/info/exclude)?
